frontend-frameworks/README.md

```diff
@@ -0,0 +1,47 @@
+# Frontend Frameworks
+
+This repo contains some skeletons for different frontend frameworks you can leverage from you Lucid editor extension.
+
+## Getting started
+To make changes to the application, you will need to install the dependencies and run the development server.
+
+Install dependencies for the React app first:
+
+```bash
+/frontend-frameworks/editorextensions/react-example/react-example$ npm install
+```
+
+Then install the dependencies for the editor extension:
+
+```bash
+/frontend-frameworks/editorextensions/react-example$ npm install
+```
+
+Then you can run the development server, this will also start any frontend framework development frameworks in the package that have an `npm start` script defined:
+```bash
+/frontend-frameworks$ npm start
+```
+
+## Folder / file structure
+
+The code is organized into the following folders:
+```
+frontend-frameworks
+├── editorextensions
+│   └── react-example
+│       ├── react-example
+│       │   ├── public
+│       │   └── src
+│       ├── resources
+│       └── src
+├── public
+│   └── react-example
+└── shapelibraries
+```
+
+`/editorextensions` contains the editor extensions used for the integration.
+There is one editor extension per frontend framework.
+
+`/public` contains the files which will be statically hosted alongside the editor extensions, and available through relative urls.
+For frontend frameworks during development, the public folder will typically include a `.html` file which links to the locally running server.
+When building and deploying your extension for production, you can put the whole production bundle for your frontend application in this folder and it will be statically hosted and available for use by your extension.
```

Comment:
When i do a production build in react, we get this 
![image](https://github.com/lucidsoftware/sample-lucid-extensions/assets/81441553/7ce57428-30ef-4a14-9465-93022e1ab938)


can we also possible what should be provided here? I think it wasn't obvious and remember finding this out by trial and error.

Reply:
I think this is a good followup. We need to have a better thought about how we instruct people to do deploys that I think warrants its own documentation page, etc. I'm going to hold off for now but totally agree we should do this